api/v1/podconfig_types.go

```diff
@@ -0,0 +1,61 @@
+package v1
+
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// +kubebuilder:rbac:groups=k8up.io,resources=podconfigs,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=k8up.io,resources=podconfigs/status;snapshots/finalizers,verbs=get;update;patch
```

Comment:
have You tested it in Openshift? It might be necessary to put create for /finalizers

Reply:
The OpenShift bug you're referring is very specific for job/cronjobs and the used kubectl version :)

Also, the operator itself currently only reads the `PodConfig` types. We could actually remove quite a few of the permissions here.